Brownie-Points.md

```diff
@@ -2,12 +2,11 @@ The number beside your picture on Free Code Camp tells you how many Brownie Poin
 
 ![A user profile picture next to a with Brownie Points score](http://i.imgur.com/SsvbkDH.png)
 
-There are three ways you can get Brownie Points:
+There are two ways you can get Brownie Points:
 
 1. Complete challenges - you get one point per challenge you complete    
-1. Post relevant links on [Camper News](Camper-News) - you get 1 point for posting a link, and another point for each upvote your link gets    
 1. Help other campers in chat - each time you help another camper and they thank you (by typing "thanks @yourname"), you will get a point    

```

Comment:
@abisz sure. Indeed that's the way. Just the numbering in the markdown text can be written in incremental order. Although, markdown renderers will show it in proper order even if numbering is messed up but just for consistency, it is good to write it in the following way...

```md
1. Item 1
2. Item 2
3. Item 3
```

Ref. [**Markdown Guide**](https://guides.github.com/features/mastering-markdown/)